server/src/main/java/org/elasticsearch/action/admin/indices/datastream/GetDataStreamsAction.java

```diff
@@ -72,26 +73,26 @@ public ActionRequestValidationException validate() {
 
         public Request(StreamInput in) throws IOException {
             super(in);
-            this.names = in.readStringArray();
+            this.name = in.readString();
         }
 
         @Override
         public void writeTo(StreamOutput out) throws IOException {
             super.writeTo(out);
-            out.writeStringArray(names);
+            out.writeString(name);
```

Comment:
I think there is a BWC issue here since the original code is in the 7.7 branch? Probably easiest to just make it 7.7 compatible here (and in the read above) rather than other workarounds.